mobile/Components/MindBodyCard.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {
+  View, Text,
+} from 'react-native';
+import PropTypes from 'prop-types';
+
+export default function MindBodyCard({ activity, duration, boxWidth }) {
+  return (
+    <View
+      style={{
```

Comment:
Try to avoid inline styling when possible! React Native's StyleSheet.create caches styles super efficiently (`styles.container` in `HomeScreen` for example). But if you declare the styles as inline then during every render, your styles are sent over the bridge (which can lessen performance)!